src/data/zwave.ts

```diff
@@ -16,3 +23,8 @@ export const fetchNetworkStatus = (
   hass.callWS({
     type: "zwave/network_status",
   });
+
+export const fetchValues = (
+  hass: HomeAssistant,
+  nodeId: number
+): Promise<ZWaveValue[]> => hass.callApi("GET", `zwave/values/${nodeId}`);
```

Comment:
This typing is a little more easier to read:

```suggestion
) => hass.callApi<ZWaveValue[]>("GET", `zwave/values/${nodeId}`);
```

(don't just commit this suggestion, as prettier will restructure)